app/helpers/compute_resources_vms_helper.rb

```diff
@@ -12,11 +12,12 @@ def vm_power_actions(host, vm)
   end
 
   def vm_console(host, vm)
+    options = { :class => "btn btn-info", :id => "console-button" }
```

Comment:
`id` has to be unique on the page, if there are multiple vms (e.g. on the compute resource vm list), this will not work. I suggest either making it include some unique number if we need to access a specific button (e.g. `console-button-#{vm.id}`) or change it to part of the class attribute, which can have multiple instances of the same class (i.e. `class: "btn btn-info console-button"`)
The same comment is true also for the other function below.

Reply:
On my instance I saw the button only on details pages. The one on `/hosts/$name` is controlled by `vm_console` and the one on `/compute_resources/$cr_id-$cr_name/vms/$vm_uuid` is controlled by `vm_console_action`. In vm list I'm getting only power actions, import as (un)?managed host and delete, I'm not getting any console there.

Reply:
@tbrisker is your concern resolved?